templates/desktop/ubuntu-kylin.html

```diff
@@ -6,72 +6,93 @@
-    		<p>The team is working together with the open-source community to adapt the core features of Ubuntu to better suit Chinese users. Along with localised versions of the Ubuntu Software Centre, Ubuntu Kylin also features some of the most commonly used Chinese applications.</p>
-    	</div>
-    	<div class="six-col text-center last-col">
-        	<ul class="no-bullets clearfix align-center">
-        		<li><img src="{{ ASSET_SERVER_URL }}e9ec49b2-logo-canonical-288.png" alt="Canonical" width="288" height="40" /></li>
-        		<li><img src="{{ ASSET_SERVER_URL }}b8b1a85b-logo-csip-180.png" alt="CSIP" width="180" height="71" /></li>
-        		<li><img src="{{ ASSET_SERVER_URL }}3c62e9e2-logo-nudt-288.png" alt="NUDT" width="288" height="72" /></li>
-        	</ul>
-    	</div>
+        <div class="six-col">
+            <p>To create Ubuntu Kylin, Canonical partnered with China Software and Integrated Chip Promotions Centre (CSIP) and the National University of Defense Technology (NUDT) to produce a fully localised and customised version of Ubuntu.</p>
+            <p>The team is working together with the open-source community to adapt the core features of Ubuntu to better suit Chinese users. Along with localised versions of the Ubuntu Software Centre, Ubuntu Kylin also features some of the most commonly used Chinese applications.</p>
+        </div>
+        <div class="six-col text-center last-col">
+            <ul class="no-bullets clearfix align-center">
+                <li><img src="{{ ASSET_SERVER_URL }}e9ec49b2-logo-canonical-288.png" alt="Canonical" width="288" height="40" /></li>
+                <li><img src="{{ ASSET_SERVER_URL }}b8b1a85b-logo-csip-180.png" alt="CSIP" width="180" height="71" /></li>
+                <li><img src="{{ ASSET_SERVER_URL }}3c62e9e2-logo-nudt-288.png" alt="NUDT" width="288" height="72" /></li>
+            </ul>
+        </div>
     </div>
 </section>
 
 <section class="row row-grey">
     <div class="strip-inner-wrapper">
-    	<h2>What&rsquo;s new in Ubuntu {{latest_release_full}}?</h2>
-    
-    	<p class="eight-col">Ubuntu Kylin {{latest_release_full}} is the latest LTS release. This release has brand new user guidance to help first time users to navigate around the desktop environment and WeChat, the popular messaging app has been added.</p>
-    	<div class="combined-list">
-    		<ul class="list-ticks six-col">
-     			<li>Updated kernel to 4.4 LTS version</li>
-     			<li>New first time user guide</li>
-    		</ul>
-    		<ul class="list-ticks six-col last-col">
-     			<li>Support for the ARM64 chipset</li>
-     			<li>WeChat web application</li>
-    		</ul>
-    	</div>
-    	<div class="eight-col">
-     		<p><a href="https://wiki.ubuntu.com/{{latest_release_name}}/ReleaseNotes/UbuntuKylin" class="external">Read the full release notes</a></p>
+        <h2>What&rsquo;s new in Ubuntu {{lts_release_no_point}}?</h2>
```

Comment:
@anthonydillon There's a variable for that which I've now changed that to - `{{lts_release_full}}`